wordpress-mobile/gutenberg-mobile
# Remove the border-left from the toolbar

Fixes #639 

Gutenberg side PR: https://github.com/WordPress/gutenberg/pull/20881

With the left border | Without the left border
-- | --
<img width="75" alt="Screenshot 2020-03-13 at 17 01 35" src="https://user-images.githubusercontent.com/1032913/76633095-dc875d00-654c-11ea-8806-5d139ed3b9ef.png"> | <img width="75" alt="Screenshot 2020-03-13 at 17 01 10" src="https://user-images.githubusercontent.com/1032913/76633070-d5f8e580-654c-11ea-8db1-7b40f230c448.png">

To test:
1. Run the demo app and capture a screenshot of the app
2. Notice that there is no 1-pixel grey border to the left of the Inserter button

cc @iamthomasbishop in case you also want to try this out.

PR submission checklist:
